arviz/plots/ppplot.py

```diff
@@ -0,0 +1,181 @@
+"""Posterior-Prior plot."""
+
+from .plot_utils import (
+    xarray_var_iter,
+    _scale_fig_size,
+    get_plotting_function,
+)
+from ..utils import _var_names, get_coords
+from ..rcparams import rcParams
+
+
+def plot_pp(
+    data,
+    kind="latent",
+    figsize=None,
+    textsize=None,
+    var_names=None,
+    coords=None,
+    transform=None,
+    legend=True,
+    ax=None,
+    prior_kwargs=None,
+    posterior_kwargs=None,
+    observed_kwargs=None,
+    backend=None,
+    backend_kwargs=None,
+    show=None,
+):
+    """
+    Plot for posterior/Prior.
+
+    Parameters
+    ----------
+    data : az.InferenceData object
+        InferenceData object containing the posterior/prior data.
+    kind : str
+        kind of plot to display {"latent", "observed"}, defaults to 'latent'.
+        "latent" includes {"prior", "posterior"} and "observed" includes
+        {"observed_data", "prior_predictive", "posterior_predictive"}
+    figsize : tuple
+        Figure size. If None it will be defined automatically.
+    textsize: float
+        Text size scaling factor for labels, titles and lines. If None it will be
+        autoscaled based on figsize.
+    var_names : str, list, list of lists
+        if str, plot the variable. if list, plot all the variables in list
+        of all groups. if list of lists, plot the vars of groups in respective lists.
+    coords : dict
+        Dictionary mapping dimensions to selected coordinates to be plotted.
+        Dimensions without a mapping specified will include all coordinates for
+        that dimension.
+    transform : callable
+        Function to transform data (defaults to None i.e. the identity function)
+    legend : bool
+        Add legend to figure. By default True.
+    ax: axes, optional
+        Matplotlib axes: a numpy 2d array of matplotlib axes. Returned object will
+        have shape (nvars, 3) where the last column is the combined plot and the
+        first columns are the single plots.
+    prior_kwargs : dicts, optional
+        Additional keywords passed to `arviz.plot_kde` for prior/predictive groups.
+    posterior_kwargs : dicts, optional
+        Additional keywords passed to `arviz.plot_kde` for posterior/predictive groups.
+    observed_kwargs : dicts, optional
+        Additional keywords passed to `arviz.plot_kde` for observed_data group.
+    backend: str, optional
+        Select plotting backend {"matplotlib","bokeh"}. Default "matplotlib".
+    backend_kwargs: bool, optional
+        These are kwargs specific to the backend being used. For additional documentation
+        check the plotting method of the backend.
+    show : bool, optional
+        Call backend show function.
+
+    Returns
+    -------
+    axes : matplotlib axes or bokeh figures
```

Comment:
What's the return type more specifically? Is it a numpy array of axes or figures?  It can't be a single `matplotlib.axes.Axes`, right?

Reply:
Ah, I see -- the return value is actually described in the description of the `ax` argument.  Suggest you move (or copy) that material down to "Returns", and change the parameter description to read "The ``ax`` argument should have shape ``(nvars, 3)``, where the last column is for the combined before/after plots and columns 0 and 1 are for the before and after plots, respectively."